assets/helpers/legal.js

```diff
@@ -2,24 +2,38 @@
 
 // ----- Imports ----- //
 
-import type { IsoCountry } from 'helpers/internationalisation/country';
-
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
 
 // ----- Terms & Conditions ----- //
 
 const termsLinks: {
-  [IsoCountry]: string,
+  [CountryGroupId]: string,
 } = {
-  GB: 'https://www.theguardian.com/info/2016/apr/04/contribution-terms-and-conditions',
-  US: 'https://www.theguardian.com/info/2016/apr/07/us-contribution-terms-and-conditions',
+  GBPCountries: 'https://www.theguardian.com/info/2016/apr/04/contribution-terms-and-conditions',
+  UnitedStates: 'https://www.theguardian.com/info/2016/apr/07/us-contribution-terms-and-conditions',
+  AUDCountries: 'https://www.theguardian.com/info/2016/apr/04/contribution-terms-and-conditions',
+  EURCountries: 'https://www.theguardian.com/info/2016/apr/04/contribution-terms-and-conditions',
+  International: 'https://www.theguardian.com/info/2016/apr/04/contribution-terms-and-conditions',
+  NZDCountries: 'https://www.theguardian.com/info/2016/apr/04/contribution-terms-and-conditions',
+  Canada: 'https://www.theguardian.com/info/2016/apr/04/contribution-terms-and-conditions',
 };
 
 const privacyLink = 'https://www.theguardian.com/help/privacy-policy';
 
 const copyrightNotice = `\u00A9 ${(new Date()).getFullYear()} Guardian News and Media Limited or its
   affiliated companies. All rights reserved.`;
 
-const contributionsEmail = 'mailto:contribution.support@theguardian.com';
+const contributionsEmail: {
+    [CountryGroupId]: string,
+} = {
+  GBPCountries: 'mailto:contribution.support@theguardian.com',
+  UnitedStates: 'mailto:contribution.support@theguardian.com',
+  AUDCountries: 'mailto:apac.help@theguardian.com',
+  EURCountries: 'mailto:contribution.support@theguardian.com',
+  International: 'mailto:contribution.support@theguardian.com',
+  NZDCountries: 'mailto:contribution.support@theguardian.com',
+  Canada: 'mailto:contribution.support@theguardian.com',
+};
```

Comment:
Maybe to avoid re-typing the address we could extract it into a common variable: 

```
const defaultContributionsEmail = 'mailto:contribution.support@theguardian.com';

const contributionsEmail: {
    [CountryGroupId]: string,
} = {
  AUDCountries: 'mailto:apac.help@theguardian.com', 
  GBPCountries: defaultContributionsEmail,
  UnitedStates: defaultContributionsEmail,
  EURCountries: defaultContributionsEmail,
  International: defaultContributionsEmail,
  NZDCountries: defaultContributionsEmail,
  Canada: defaultContributionsEmail,
};
```

Reply:
done :D